lib/manageiq/environment.rb

```diff
@@ -54,7 +54,9 @@ def self.update_ui_thread
     end
 
     def self.install_bundler(root = APP_ROOT)
-      system!("gem install bundler -v '#{bundler_version}' --conservative") unless ENV["GITHUB_ACTIONS"]
+      return if system("which bundle", [:out, :err] => "/dev/null") && system("bundle info bundler", [:out, :err] => "/dev/null")
```

Comment:
I'll add a comment here since it's not obvious what the `bundle info bundler` is doing...